### PR TITLE
Selectable evaluation metric

### LINKED ISSUE #110: Plot forward selection with another evaluation metric
# Task Title

Task: Plot forward selection with another evaluation metric

## Task Description

Business user wants to see forward selection plot computed with the MAE evaluation metric.

### PR DESCRIPTION
**Pull request by sandervh14**
# Story Title

[Plot forward feature selection with another metric](https://github.com/PythonPredictions/cobra/issues/110)

## Changes made

Added an argument "metric" to 3 functions involved, allowing the developer to pass a function(y_true, y_pred) like those provided by sklearn.

## How does the solution address the problem

see above

## Linked issues

Closes #110

### CODE REVIEW

**Comment by sandervh14** [comment]
**Attention: I think it's best to only merge this pull request AFTER PR #109 General fixes bins.**
This is because this PR is based on the code from that PR, rather than the Development branch.

**Comment by sborms** [comment]
Nice solution, Sander! Will test it out but very handy approach!
